apps/remix-ide-e2e/src/tests/generalSettings.test.ts

```diff
@@ -18,13 +18,18 @@ module.exports = {
   'Should activate `generate contract metadata`': function (browser) {
     browser.waitForElementVisible('*[data-id="remixIdeSidePanel"]', 5000)
       .waitForElementVisible('*[data-id="settingsTabGenerateContractMetadataLabel"]', 5000)
+      .execute(function () {
+        const elem = document.getElementById('generatecontractmetadata') as HTMLInputElement
+        elem.checked = true
```

Comment:
By default the generate metadata checkbox should be checked. Is there any need to check it again in tests?